ExpensesTracker/app/src/main/res/layout/initial_activity.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="79dp"
+        android:layout_height="27dp"
+        android:layout_marginTop="132dp"
+        android:autoSizeTextType="uniform"
+        android:text="Όνομα"
```

Comment:
- As a good practise, text of different views should be included in the strings.xml file and be used from there
- Also we will be keeping the text to English for this app for now